src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Performance/PreferIsEmptyOverCountTests.cs

```diff
@@ -435,6 +435,22 @@ public Task CSharpTestWhitespaceTrivia()
         ? 0 :
         _dictionary.Count;
 }");
+
+        [Theory]
+        [InlineData("System.ReadOnlyMemory")]
+        [InlineData("System.ReadOnlySpan")]
+        [InlineData("System.Memory")]
+        [InlineData("System.Span")]
+        public Task CSharpTest_DisallowedTypesForCA1836_NoDiagnosis(string type)
+            => VerifyCS.VerifyAnalyzerAsync(
```

Comment:
Is it worth having the same test for vbnet?

Reply:
These types are not supported on VB (yet?).
https://github.com/dotnet/vblang/issues/297

Reply:
Oh! I thought that types were always available for all languages of the .net stack.